README.md

```diff
@@ -121,6 +122,36 @@ Custom translations of properties can be added by extending the
       (println rows)))
   (close-datasource datasource))
 ```
+### Neo4j 
+
+```clojure
+(ns hikari-cp.example
+  (:require [hikari-cp.core :refer :all]
+            [clojure.java.jdbc :as jdbc]))
+
+(def datasource-options {:adapter            "neo4j:bolt" ; neo4j:bolt+routing
+                         :username           "username"
+                         :password           "password"
+                         :port-number        7687
+                         :withMaxConnectionLifetime 1800000 
+                         :withMaxConnectionPooolSize 50
+                         :withConnectionAcquisitionTimeout 120000 
+                         })
+
+
+;; OR explicitly
+(def datasource-options {:jdbc-url "jdbc:neo4j:bolt://host:port/?username=neo4j,password=xxxx,debug=true" })
+
+(def datasource
```

Comment:
Fits one line?

Reply:
`jdbc-url` connection string fits in one line, with all parameters passed in.